libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/LoggingValueSerializer.scala

```diff
@@ -25,6 +36,34 @@ private[logging] object LoggingValueSerializer {
         generator.writeStartArray()
         sequence.foreach(writeValue(_, generator))
         generator.writeEndArray()
+      case LoggingValue.OfJson(jsValue) =>
+        def write(jsValue: JsValue): Unit =
+          jsValue match {
+            case JsNull =>
+              generator.writeNull()
+            case JsTrue =>
+              generator.writeBoolean(true)
+            case JsFalse =>
+              generator.writeBoolean(false)
```

Comment:
```suggestion
```

Reply:
Actually tried that already but the compiler complained ._.

Reply:
iirc it breaks exhaustiveness checks since `JsBoolean` is an `unapply` thingy.

Reply:
If that's true, then the `JsBoolean` case should be removed. Unless we are defensively programming for the addition of a third boolean value in the future 👻